server.js

```diff
@@ -1,26 +1,89 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import listEndpoints from "express-list-endpoints";
+import dotenv from "dotenv";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happy-thoughts-api";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+//Get all messages
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+//Create model for mongo database
+const HappyThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlenght: 5,
+    maxlenght: 140,
+    trim: true, 
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now()
+  }
+})
+
+const Thought = mongoose.model("Thought", HappyThoughtSchema);
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndpoints(app));
+});
+
+//Show thoughts
+app.get("/thoughts", async (req, res) => {
+  try {
+    const thoughts = await Thought.find({})
+      .sort({ createdAt: -1 }).limit(20).exec()
+    res.status(200).json({ success: true, response: thoughts });
+  } catch (error) {
+    res.status(400).json({ success: false, response: error });
```

Comment:
The very small and only suggestion I have is to maybe give a more descriptive error. :)